css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
+	border: $border-width-md solid $border !important;
+	border-radius: $border-radius-lg !important;
+	border-color: transparent !important;
+	background: linear-gradient(90deg, $gradient-vivid-start, $gradient-vivid-end) border-box !important;
+
+	.gradient-card-content {
+		border-radius: $border-radius !important;
```

Comment:
Should some default padding be applied to the `gradient-card-content`?

Reply:
Makes sense, confirmed the default can be overridden by atomics. Defaulted to padding-xs as it's mostly used for banners in the Copilot AI branding